.travis.yml

```diff
@@ -1,21 +1,16 @@
 language: python
 sudo: false
-python: 2.7
+python:
+    - 2.7
+    - 3.6
 env:
-    - TOX_ENV=flake8
-    - TOX_ENV=py27-django18
-    - TOX_ENV=py34-django18
-    - TOX_ENV=py27-django19
-    - TOX_ENV=py34-django19
-    - TOX_ENV=py27-djangomaster
-    - TOX_ENV=py34-djangomaster
+    - DJANGO=1.8
+    - DJANGO=1.11
```

Comment:
I actually would like to keep the Django 1.9 testing in Travis, since that is a version we state that we support--I put priority of running the tests against supported versions over more recent versions that we don't support and are allowing failures on.

Reply:
ah, small misunderstanding. I'm on it!